BowlingBall.Tests/TheBowlingGameTest.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
```

Comment:
Why you have added 'The' in class name????
Also rename it to 'BowlingGameFixture'